pkgs/applications/editors/kakoune/plugins/openscad.kak.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "openscad.kak";
+  version = "unstable-2020-11-16";
```

Comment:
If I get it right https://nixos.org/manual/nixpkgs/stable/#sec-package-naming
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

It should be:
```nix
    name = "openscad.kak-unstable";
    version = "2020-11-16";
```

Reply:
The way I had it appears to be more common (but not always the case):

```
jfelice@C02CW0J5ML87 nixpkgs (415130beb4b)
$ git grep 'version = "unstable-' |wc -l
     347

jfelice@C02CW0J5ML87 nixpkgs (415130beb4b)
$ git grep 'pname = .*-unstable"' |wc -l
     120

```